pom.xml

```diff
@@ -383,9 +383,9 @@
                 <version>1.1.1</version>
             </dependency>
             <dependency>
-                <groupId>com.hpe.caf.externals</groupId>
```

Comment:
Couldn't find this in our code base, so replaced it with https://github.com/joestelmach/natty/releases/tag/natty-0.13

Reply:
It is [here](https://gitlab.otxlab.net/cybersecurity-enterprise/cddri/caf/natty-date-parser).

Reply:
Is it ok to use this 'com.hpe.caf.externals' in the opensource pom ? Or should we refactor it to 'com.github.caf.externals' and use it here ?

Reply:
The reliance on a closed-source module is a pre-existing issue so we don't need to tackle it as part of this ticket.  (It only builds because `USE_OPEN_SOURCE_SETTINGS=false` is set in the `official-build.props` file.)  For this let's just release a new version of the natty-date-parser fork with a different group id and then update this to use it.